src/db/nedb.js

```diff
@@ -13,14 +14,22 @@ const db = {
 
 // Init datastores
 async function initDB() {
-  const basePath = `${Utils.getDataDir()}/nedb`;
-  logger.info(`DB path: ${basePath}`);
+  try {
+    await migrateDB();
```

Comment:
directly call `await migrateTxDB()`? also, you are awaiting a non-async func

Reply:
Leave then structure here for more migration script in the future
add `async` keyword

Reply:
Leave then structure here for more migration script in the future
add async keyword